src/mail/view/MailViewer.js

```diff
@@ -1724,7 +1722,7 @@ export class MailViewer {
 			     .some(s => (s.color && s.color !== "inherit") || (s.backgroundColor && s.backgroundColor !== "inherit"))
 			|| html.querySelectorAll('font[color]').length > 0
 
-		this._sanitizedMailBody = m.trust(stringifyFragment(html))
+		this._sanitizedMailBody = m.trust(await worker.urlify(stringifyFragment(html)))
```

Comment:
I just noticed that we save vnode. this is **bad** don't do this (result of m.trust()).